src/parser.cpp

```diff
@@ -479,11 +481,129 @@ options_metadata parser::load_settings()
         "node.refresh_transactions",
         value<bool>(&configured.node.refresh_transactions),
         "Request transactions on each channel start, defaults to false."
+    )
+
+    /* [bitcoin] */
+    (
+        "bitcoin.retargeting_factor",
+        value<uint32_t>(&configured.bitcoin.retargeting_factor)->notifier(
+            dependent_config_setter),
+        "The retargeting factor, defaults to 4."
+    )
+    (
+        "bitcoin.target_spacing_seconds",
+        value<uint32_t>(&configured.bitcoin.target_spacing_seconds)->notifier(
+            dependent_config_setter),
+        "The target block spacing time in seconds, defaults to 600."
+    )
+    (
+        "bitcoin.easy_spacing_seconds",
+        value<uint32_t>(&configured.bitcoin.easy_spacing_seconds),
+        "The easy target block spacing time in seconds, defaults to 1200."
```

Comment:
Shouldn’t this have been removed?

Reply:
No, to my understanding both values, `target_spacing_seconds` and `easy_spacing_seconds` are used by testnet.

Reply:
> If not I'd like to just remove easy spacing

:+1: 

> I'm tempted to do the same with `retargeting_factor` if we don't know of common coins that are changing it.

There are hard forks in ftc that are changing it. But the initial value remains 4.

> These might be clearer as:

    block_spacing_seconds (10 min blocks)
    timestamp_limit_seconds (2 hour future clock limit)
    retargeting_interval_seconds (2 week targeting interval)

Are you suggesting to add the text in the parenthesis to the help text's mention of the default value instead of the one in seconds?

Reply:
No

Reply:
Done https://github.com/libbitcoin/libbitcoin/pull/1025

Reply:
> If not I'd like to just remove easy spacing

Done https://github.com/libbitcoin/libbitcoin/pull/1028